Browser_IDE/index.html

```diff
@@ -2,36 +2,135 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>SplashKit Online</title>
-
-    <link rel="stylesheet" href="codemirror-5.65.15/lib/codemirror.css">
-    <link rel="stylesheet" href="codemirror-5.65.15/theme/dracula.css">
-    <link rel="stylesheet" href="codemirror-5.65.15/addon/hint/show-hint.css">
-    <link rel="stylesheet" href="codemirror-5.65.15/addon/fold/foldgutter.css">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
-    <link rel="stylesheet" href="stylesheet.css">
-
-    <script src="codemirror-5.65.15/lib/codemirror.js"></script>
-    <script src="jszip/jszip.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"
-        integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3"
-        crossorigin="anonymous"></script>
-    <script src="codemirror-5.65.15/mode/javascript/javascript.js"></script>
-    <script src="codemirror-5.65.15/addon/edit/closebrackets.js"></script>
-    <script src="codemirror-5.65.15/addon/selection/active-line.js"></script>
-    <script src="codemirror-5.65.15/addon/hint/show-hint.js"></script>
-    <script src="splashkit-javascript-hint.js"></script>
-    <script src="codemirror-5.65.15/addon/fold/foldcode.js"></script>
-    <script src="codemirror-5.65.15/addon/fold/foldgutter.js"></script>
-    <script src="codemirror-5.65.15/addon/fold/brace-fold.js"></script>
-    <script src="codemirror-5.65.15/addon/fold/comment-fold.js"></script>
-    <script src="split.js/split.min.js"></script>
-    <script type="module">import mime from "./mime/src/index.js"; window.mime = mime;</script>
-	
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>SplashKit Online</title>
+
+	<link rel="stylesheet" href="codemirror-5.65.15/lib/codemirror.css">
+	<link rel="stylesheet" href="codemirror-5.65.15/theme/dracula.css">
+	<link rel="stylesheet" href="codemirror-5.65.15/addon/hint/show-hint.css">
+	<link rel="stylesheet" href="codemirror-5.65.15/addon/fold/foldgutter.css">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+	<link rel="stylesheet" href="stylesheet.css">
+
+	// adding fav icon
+	<link rel="icon" href="https://thoth-tech.github.io/SplashkitOnline/favicon.ico" />

```

Comment:
This does work, but it'd be much better if it was still a relative path in case we change where we host SplashKit Online. `<link rel="icon" href="favicon.ico">` works fine :yum: